### PR TITLE
Pr uirefinement

### PR DESCRIPTION
**Pull request by janefwp**
nearly finished except breakdown component

### CODE REVIEW

**Comment by janefwp** [comment]
one question:
if I use 
---------------------------
<React.StrictMode>
    <App />
  </React.StrictMode>,
---------------------------
there will be a below error :
----------------------------
index.js:1 Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of Transition which is inside StrictMode. Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here
---------------------------------------------------------------------------------------------------------------------------

I don't know how to solve this problem.

**Comment by whywhy36** [comment]
> ## one question:
> if I use
> ## <React.StrictMode>
> 
> </React.StrictMode>,
> ## there will be a below error :
> ## index.js:1 Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of Transition which is inside StrictMode. Instead, add a ref directly to the element you want to reference. Learn more about using refs safely here
> I don't know how to solve this problem.

FYI: https://github.com/jossmac/react-toast-notifications/issues/82

This is an issue in the used library. Follow the issue thread to use the workaround or choose a different lib.

**Comment by janefwp** [comment]
use "react-hot-toast" and fixed this issue

**Comment by janefwp** [comment]
finished this task include component breakdown.

File: src/screens/HomeScreens/components/InfoTable/InfoTable.js [hunk 1]
```diff
@@ -1,78 +1,157 @@
...
                             <td>{item.description}</td>
-                            </LinkContainer>
+                            </LinkContainer>  
                             <td>{item.category}</td>
+                            
+                            <td>{item.deadline.toDateString()}</td>
                            <td><Button variant="light" onClick={() => deleteInfoHandler(item.id)}>Delete</Button></td>
-                        </tr>
-                
+                        </tr>
```
Comment (whywhy36) [inline src/screens/HomeScreens/components/InfoTable/InfoTable.js]: can this `<tr>` be extracted into a separate component?

File: src/screens/HomeScreens/components/InfoTable/InfoTable.js [hunk 2]
```diff
@@ -1,78 +1,157 @@
...
+            <br />
+            {(infos.length!=0) && 
+            <div>
+            <Row>
+                <Col md={4}>
+                    <Button variant='secondary' disabled={(checkNum===0)? true : false} onClick={deleteSelectedInfoHandler}>Delete seleted</Button> 
+                </Col>
+                <Col md={{ span: 4, offset: 4 }}>
+                    <label >Filter by category: &nbsp;</label>
+                    <select name="category" onChange={filterHandler}>
```
Comment (whywhy36) [inline src/screens/HomeScreens/components/InfoTable/InfoTable.js]: I could see duplicated codes for this select options. Consider consolidating this part with `TodoSelect.js`

File: src/screens/HomeScreens/components/InfoTable/InfoTable.js [hunk 3]
```diff
@@ -1,78 +1,157 @@
...
+        if(checkNum ===0){
+            setAllChecked(false)
+        }
+        else {
+            if(checkNum===infos.length){
+                setAllChecked(true)
+            }
+            else {
+                setAllChecked(false)
+            }
```
Comment (whywhy36) [inline src/screens/HomeScreens/components/InfoTable/InfoTable.js]: `setAllChecked(checkNum === infos.length)`

Comment (janefwp) [reply src/screens/HomeScreens/components/InfoTable/InfoTable.js]: done

File: src/screens/HomeScreens/components/InfoTable/InfoTable.js (hunk 2 above)
Comment (janefwp) [reply src/screens/HomeScreens/components/InfoTable/InfoTable.js]: done

File: src/screens/HomeScreens/components/InfoTable/InfoTable.js (hunk 1 above)
Comment (janefwp) [reply src/screens/HomeScreens/components/InfoTable/InfoTable.js:148]: In this session ,there are 2 event (onChange and onClick). how to use these events in one Object tag? like<Infolist onChange={} onClick={} />?

Comment (janefwp) [reply src/screens/HomeScreens/components/InfoTable/InfoTable.js]: done

**Comment by janefwp** [comment]
Can I merge this branch to main now?

**Review by whywhy36 — APPROVED** [review]

**Pull request merged**